gson/src/main/java/com/google/gson/Gson.java

```diff
@@ -106,12 +106,13 @@
  * for a more complete set of examples.</p>
  *
  * <h2>Lenient JSON handling</h2>
- * For legacy reasons most of the {@code Gson} methods allow JSON data which does not
+ * For legacy reasons most of the {@code Gson} methods by default allow JSON data which does not
  * comply with the JSON specification, even when the strictness {@link Strictness#DEFAULT}
  * is used. To restrict a {@code Gson} instance from parsing JSON that
- * does not comply with the JSON specification, {@linkplain Strictness#STRICT strict} mode should be used.
- * Alternatively, if you do not want to use {@linkplain Strictness#STRICT strict} mode,
- * the following workarounds can be used:
+ * does not comply with the JSON specification, {@link Strictness#STRICT} mode should be used.
+ *
+ * <p>For older Gson versions which don't have the {@link Strictness} mode API the following
+ * workarounds can be used:
```

Comment:
This section probably needs to be rewritten a bit once `GsonBuilder.setStrictness` affects `Gson.fromJson` and `Gson.toJson` (see https://github.com/google/gson/pull/2323#issuecomment-1500961215), but that can be done later I guess when the behavior is actually changed.